### DISCUSSION TITLE
v2.0

### DISCUSSION THREAD

**Discussion by port19x** [pystardust/ani-cli#612] [General] [answered]
## What's Changed
_In the interest of time, the following patchnotes aren't comprehensive_

- Documentation
  - Major refactors by @iamchokerman (#584)
- Bugfixes
  - cryptography issue by @CoolnsX (#581)
  - new gogo keys by @justfoolingaround (#601)
- Features
  - new ux by @iamchokerman (#584)
  - way faster history parsing by @CoolnsX (#584)
  - rapid resume by @71zenith (#573)
  - trackma support by @iamchokerman (#581)

## New Contributors
* @justfoolingaround made their first contribution in https://github.com/pystardust/ani-cli/pull/601

**Full Changelog**: https://github.com/pystardust/ani-cli/compare/v1.9...v2.0

<hr /><em>This discussion was created from the release <a href='https://github.com/pystardust/ani-cli/releases/tag/v2.0'>v2.0</a>.</em>

**Comment by greqov** [comment]
There is no "select quality" option on v2.0? Oh I've missed something?

**Reply by justchokingaround** [reply, marked as answer]
we removed it in the menu, but you can still pass it as an argument when running ani-cli -q 720p for example
